holidays/countries/chile.py

```diff
@@ -110,20 +101,14 @@ def _get_movable(hol_date: date) -> date:
         # Naval Glories Day (Law 2.977).
         self._add_holiday(tr("Día de las Glorias Navales"), MAY, 21)
 
-        # National Day of Indigenous Peoples.
-        name = tr("Día Nacional de los Pueblos Indígenas")
-        if year == 2021:
-            self._add_holiday(name, JUN, 21)
-        elif year >= 2022:
-            epoch = Sun.get_equinox_solstice(year, target="summer")
-            # Received date for UTC timezone needs to be adjusted
-            # to match Chile's timezone
+        if year >= 2021:
+            # National Day of Indigenous Peoples.
+            name = tr("Día Nacional de los Pueblos Indígenas")
             # https://www.feriadoschilenos.cl/#DiaNacionalDeLosPueblosIndigenasII
-            equinox = map(int, Epoch(epoch).get_full_date())
-            adjusted_date = datetime(*equinox, tzinfo=timezone.utc).astimezone(
-                ZoneInfo("America/Santiago")
-            )
-            self._add_holiday(name, JUN, adjusted_date.day)
+            if year == 2021:
+                self._add_holiday(name, JUN, 21)
+            else:
+                self._add_holiday(name, self._summer_solstice_date(year))
```

Comment:
Must be combined with the suggestion below:
```suggestion
                self._add_holiday(name, self._summer_solstice_date)
```